experiments/ClimaEarth/cli_options.jl

```diff
@@ -2,107 +2,136 @@ import ArgParse
         default = false
-        "--turb_flux_partition"
-        help = "Method to partition turbulent fluxes. [`PartitionedStateFluxes`, `CombinedStateFluxesMOST` (default)]"
+        "--FLOAT_TYPE"
+        help = "Floating point precision  [`Float64` (default), `Float32`]"
         arg_type = String
-        default = "CombinedStateFluxesMOST"
+        default = "Float64"
+        "--device"
+        help = "Device type to use [\"auto\" (default), \"CPUSingleThreaded\", \"CPUMultiThreaded\", \"CUDADevice\"]"
+        arg_type = String
+        default = "auto"
+        # Time information
+        "--t_end"
+        help = "[REQUIRED] End time of the simulation (no default) [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        "--t_start"
+        help = "Start time of the simulation [\"0secs\" (default); allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        default = "0secs"
+        "--start_date"
+        help = "Start date of the simulation, in format \"YYYYMMDD\" [\"20100101\" (default)]"
+        arg_type = String
+        default = "20000101"
+        "--dt_cpl"
+        help = "Coupling time step in seconds [400 (default); allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        default = "400secs"
+        "--dt"
+        help = "Component model time step [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        default = "400secs"
+        "--dt_atmos"
+        help = "Atmos simulation time step (alternative to `dt`; no default) [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        "--dt_land"
+        help = "Land simulation time step (alternative to `dt`; no default) [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        "--dt_ocean"
+        help = "Ocean simulation time step (alternative to `dt`; no default) [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        "--dt_seaice"
+        help = "Sea ice simulation time step (alternative to `dt`; no default) [allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        "--dt_save_to_sol"
+        help = "Time interval for saving output [\"10days\" (default); allowed formats: \"Nsecs\", \"Nmins\", \"Nhours\", \"Ndays\", \"Inf\"]"
+        arg_type = String
+        default = "10days"
+        # Checkpointing information
         "--hourly_checkpoint"
-        help = "Boolean flag indicating whether to checkpoint at intervals of 1 hour or multiple hours"
+        help = "Boolean flag indicating whether to checkpoint at multiple-hourly intervals [false (default), true]"
         arg_type = Bool
         default = false
         "--hourly_checkpoint_dt"
```

Comment:
Could you rename this to just `checkpoint_dt` and take the same time format as the other time intervals?

Reply:
Yes and I would like to generalize this, but it requires some changes in `TimeManager` since the `HourlyCallback` and `MonthlyCallback` are hardcoded to take in number of hours/months respectively. It won't be difficult, but I'd rather do it in a follow-up PR to keep this one from getting too large

Reply:
I opened an issue for it here https://github.com/CliMA/ClimaCoupler.jl/issues/1099